### ISSUE TITLE
Use crypto.timingSafeEqual for comparison

### ISSUE THREAD

**Issue by niftylettuce** [koajs/basic-auth#17]
https://github.com/jshttp/basic-auth/issues/39

**Comment by coderhaoxin** [comment]
PR welcome :)

**Comment by niftylettuce** [comment]
Just waiting on my PR to get accepted <https://github.com/suryagh/tsscmp/pull/7> and then I will submit PR from my work already done here <https://github.com/niftylettuce/basic-auth>.

**Comment by niftylettuce** [comment]
PR made! @coderhaoxin

**Comment by niftylettuce** [comment]
@coderhaoxin can you `npm owner add niftylettuce koa-basic-auth`?

**Comment by coderhaoxin** [comment]
@niftylettuce added :)

**Comment by niftylettuce** [comment]
dude thx so much, will take care of this
